### PR TITLE
Allow seleting comments column in documents mode

### PR DESCRIPTION
**Pull request by s-l-teichmann**
Before this PR selecting the `comments` column was only allowed in `advisory` mode.
This PR allows to use in `documents` mode, too.

TODO:

- [ ] Adjust UI (Query-Editor, etc)

### CODE REVIEW

**Comment by ThomasJunk** [comment]
@JanHoefelmeyer This is ready to merge after your approval.

**Review by JanHoefelmeyer — APPROVED** [review]
LGTM & works

**Pull request merged**
